gateway/api/models.py

```diff
@@ -58,6 +58,27 @@ def __str__(self):
         return f"{self.id}"
 
 
+class Namespace(models.Model):
```

Comment:
Namespace has a [specific meaning](https://cloud.ibm.com/docs/Registry?topic=Registry-registry_overview#overview_elements_namespace) in the context of container registries, so we probably don't want to use that term here to mean something different.  

As I understand it, what we're trying to define here is more or less the author of a particular function, with links to the registry, namespace, and admin group. So something like `Provider` or `Author` might better represent the purpose of the object.

Reply:
Thanks for the feedback @psschwei ! `Author` is something that we are just using and I think it doesn't fit really well here. Probably `Provider` is a better option, I'm going to take a look and change it today 😄

Reply:
Solved in [0277a28](https://github.com/Qiskit/qiskit-serverless/pull/1331/commits/0277a280fa9a5e3c296a0c1bd37167aa7a8bc62a)